schema/cassandra/temporal/versioned/v1.8/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "CurrVersion": "1.21",
```

Comment:
Why 1.8?

Reply:
presumably it's the next sequential version?

Reply:
Yes, it's the next sequential version, I wrongly assumed this version is tied to a server release initial, this change fixes that.